cmd/reppl/main.go

```diff
@@ -38,6 +38,12 @@ func main() {
 		{
 			Name:   "eval",
 			Action: actions.Eval,
+			Flags: []cli.Flag{
+				cli.StringSliceFlag{
+					Name:  "environment, e",
```

Comment:
`repeatr run -h` says it takes `--env value, -e value`, so let's use that shorter one here as well?